go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/GoogleCloudPlatform/guest-agent
 
-go 1.13
+go 1.12
```

Comment:
can you comment on the downgrade?

Reply:
This was to match what we had elsewhere, wanted to keep the same for osconfig agent. If we want to move to 1.13 I'd be happy to but want to make sure it works for our partners

Reply:
ack